### PR TITLE
BUGFIX: TMA-384 parameterize log level by GDC_LOG_LEVEL

### PR DESCRIPTION
**Pull request by trung-le-quoc**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/493/ : FAILURE in 27s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/432/ : SUCCESS in 42m 17s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/169/ : SUCCESS in 2m 02s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/494/ : SUCCESS in 27s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/433/ : SUCCESS in 42m 38s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/170/ : SUCCESS in 1m 49s

File: lib/gooddata/bricks/middleware/logger_middleware.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ def call(params)
           logger = NilLogger.new
         else
           logger = params[:GDC_LOGGER_FILE].nil? ? Logger.new(STDOUT) : Logger.new(params[:GDC_LOGGER_FILE])
+          logger.level= params['GDC_LOG_LEVEL'] if params['GDC_LOG_LEVEL']
```
Comment (panjan) [inline lib/gooddata/bricks/middleware/logger_middleware.rb]: Please add a whitespace in front of the equals sign.

File: .rubocop.yml [hunk 2]
```diff
@@ -96,3 +96,7 @@ UnneededInterpolation:
 
 UselessAccessModifier:
   Enabled: false
+
+SpaceAroundOperators:
```
Comment (panjan) [inline .rubocop.yml]: Please don't suppress rubocop rules.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/510/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/449/ : SUCCESS in 43m 12s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/186/ : SUCCESS in 1m 48s

**Pull request merged**